Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
```

Comment:
IMO we shouln't track a Gemfile.lock in source control for a gem

Reply:
missed this, but agree